index.html

```diff
@@ -3259,16 +3241,6 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             {{BufferSource}}, [= exception/throw =] a {{TypeError}}
             and abort these steps.
           </li>
-          <li>
```

Comment:
Isn't this part of "mapping binary data to NDEF"? It is for MIME type records. Didn't you want to remove from external type (mapping external data to NDEF)?

Reply:
Good catch indeed. I was too quick. I'll fix this.